src/script/standard.h

```diff
@@ -27,16 +27,27 @@ class CScriptID : public uint160
     CScriptID(const uint160& in) : uint160(in) {}
 };
 
-static const unsigned int MAX_OP_RETURN_RELAY = 83; //!< bytes (+1 for OP_RETURN, +2 for the pushdata opcodes)
+/**
+ * Default setting for nMaxDatacarrierBytes. 80 bytes of data, +1 for OP_RETURN,
+ * +2 for the pushdata opcodes.
+ */
+static const unsigned int MAX_OP_RETURN_RELAY = 83;
+
+/**
+ * A data carrying output is an unspendable output containing data. The script
+ * type is designated as TX_NULL_DATA.
+ */
 extern bool fAcceptDatacarrier;
+
+/** Maximum size of TX_NULL_DATA scripts that this node considers standard. */
```

Comment:
I'm not sure if that's necessary to add here. This is the byte limit for OP_RETURN scripts, then `IsStandardTx` further restricts the number of OP_RETURN outputs per tx to 1. If this comment were to say `per transaction` it sort of implies that the data can be split across multiple OP_RETURN outputs as long as the limit is not exceeded.